lib/console.js

```diff
@@ -139,12 +139,14 @@ Console.prototype.dir = function dir(object, options) {
 };
 
 
-Console.prototype.time = function time(label) {
+Console.prototype.time = function time(label = 'default') {
+  label = `${label}`;
```

Comment:
Found the missing piece the `console` IDL - https://console.spec.whatwg.org/#console-namespace